packages/cli/src/commands/deploy/index.ts

```diff
@@ -194,6 +201,28 @@ export default async (client: Client) => {
       );
       return 1;
     }
+
+    const prebuiltBuild = fs.readJSONSync(
```

Comment:
`fs.readJSONSync` behaves differently from `fs.readJSON` when `throws: false` is passed in. The sync version will not throw when the file is missing, but the async version will. (They both won't throw if parsing the json in that file fails.)